assets/apps/components/src/Controls/ButtonAppearance.js

```diff
@@ -79,6 +79,13 @@ const ButtonAppearance = ({ label, value, onChange, noHover, defaultVals }) => {
 													settings[optionType]
 														.controls[controlSlug]
 												}
+												allowGradient={
```

Comment:
@preda-bogdan the Background string is translatable. If this string is translated to "Fundal", this condition won't be true.

Reply:
I think it will match the same as it will check for __( 'Fundal' ) === __( 'Fundal' )

Reply:
I find it a bit odd. If I understand correctly, you take all the controls of this type and you're checking for the label of the control, right? What if the control has the label "Overlay"?

Reply:
This control is being used just on the Button section, for the control that handles button appearance. You can see the usage. It has 2 generated controls Background and Text.

Reply:
Ok, I understand now. Thank you for your clarifications. 🚀